### ISSUE TITLE
[Java] final and classes scopes mixed in function bodies

### ISSUE THREAD

**Issue by evandrocoan** [sublimehq/Packages#871]
### [Java] final and classes scopes mixed in function bodies

![image](https://cloud.githubusercontent.com/assets/5332158/24589569/efcaa5e2-17b2-11e7-9f73-2cb147d0a8c7.png)

```java

   @Test
   public void testCalculoDoTempoDeAtrazoMaximo() throws Exception
   {
      Grafo grafo = new Grafo( true );
      final Grafo temposMaisCedo = new Grafo( true );
      final Grafo temposMaisTarde = new Grafo( true );

      final ArrayList< String > vértices = new ArrayList< >();
```

I am using the latest syntax from this repository for Java on Sublime Text 3114.

**Comment by wbond** [comment]
This is fixed by bdc2c74157ba58bf4b0189ce16ea526249c95339

**Comment by evandrocoan** [comment]
Thanks
